root/programs/C#/Frameworks/Infrastructure/CustomControl/Control/WebCustomTextBox.cs

```diff
@@ -326,12 +326,12 @@ public bool Validate(out string[] result)
             if (this.CheckType != null)
             {
                 // 必須入力チェック
-                if (this.CheckType.IsIndispensabile)
+                if (this.CheckType.IsIndispensable)
                 {
                     if ((text == ""))
                     {
                         hasError = true;
-                        lstRet.Add(CmnCheckFunction.IsIndispensabileCheckErrorMessage);
+                        lstRet.Add(CmnCheckFunction.IsIndispensableCheckErrorMessage);
```

Comment:
@yusukemaegawa 

Since this is OUT parameter, the instance of result parameter does not divert.
In addition, by using the List <string> type internally, the return value uses the string [] type.